### ISSUE TITLE
XSES: holidays are only recorded to the year 2023

### ISSUE THREAD

**Issue by ValueRaider** [gerrymanoim/exchange_calendars#295]
```
import exchange_calendars as xcal
xcal.get_calendar("XSES", start="2023-01-01", end="2024-01-01")
```

> ValueError: The XSES holidays are only recorded to the year 2023, cannot instantiate the XSES calendar through to 2024-01-01 00:00:00.

**Comment by maread99** [comment]
Hi @ValueRaider.

XSES and some other calendars rely on hardcoded holidays. If such a calendar is created with an end date beyond the date to which holidays have been included then this error is necessarily raised.

It's not a bug, just requires any user who wants the calendar to end later to offer a PR adding holidays, in this case for 2024.

Users tend to offer up these PRs towards the end of the each year.

For now, the following works:
```python
cal = xcals.get_calendar("XSES", start="2023-01-01", end="2023-12-31")
```

**Comment by MMCMA** [comment]
I don't even see 2023 for "XSES" - how can it be fixed?

```
cal = xcals.get_calendar("XSES", start="2023-01-01", end="2023-12-31")

Traceback (most recent call last):
  File "C:\ProgramData\mambaforge\envs\py310\lib\site-packages\IPython\core\interactiveshell.py", line 3460, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-9-57e402b3d4ed>", line 1, in <module>
    cal = xcals.get_calendar("XSES", start="2023-01-01", end="2023-12-31")
  File "C:\ProgramData\mambaforge\envs\py310\lib\site-packages\exchange_calendars\calendar_utils.py", line 291, in get_calendar
    return cached if cached is not None else self._fabricate(name, **kwargs)
  File "C:\ProgramData\mambaforge\envs\py310\lib\site-packages\exchange_calendars\calendar_utils.py", line 184, in _fabricate
    calendar = factory(**kwargs)
  File "C:\ProgramData\mambaforge\envs\py310\lib\site-packages\exchange_calendars\exchange_calendar.py", line 316, in __init__
    raise ValueError(self._bound_max_error_msg(end))
ValueError: The XSES holidays are only recorded to the year 2022, cannot instantiate the XSES calendar through to 2023-12-31 00:00:00.

```

**Comment by maread99** [comment]
```
$ pip install --upgrade exchange_calendars
```

**Comment by MMCMA** [comment]
version is 4.2.5 - not sure if this is the problem.

**Comment by MMCMA** [comment]
It seems I had version 4.2.4 (installed via conda-forge) - though the `print(xcals.__version__)` showed `0.0.0` not sure what is the problem with the forge package - switching and upgrading with pip solved the problem. Thanks and sorry.

**Comment by maread99** [comment]
No worries @MMCMA. We don't actively support conda (although someone might upload a version there). Glad you got it working.

**Comment by ValueRaider** [comment]
Fixed by #343
